### PR TITLE
Climber

### PR DESCRIPTION
**Pull request by kturner44**

### CODE REVIEW

**Comment by mckelveygreg** [comment]
Don't forget to rebase on master! It looks like you are quite a few commits behind 😟

File: src/main/kotlin/org/sert2521/infiniterecharge2020/climber/ClimberCommands.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import org.sert2521.sertain.coroutines.periodic
+import org.sert2521.sertain.events.onTick
+import org.sert2521.sertain.subsystems.doTask
+import org.sert2521.sertain.subsystems.use
+
+suspend fun climberUp() = doTask {
+    val climber = use<Climber>()
+    action {
+        onTick {
+            if (!DigitalInput(0).get()) {
```
Comment (will-hou) [inline src/main/kotlin/org/sert2521/infiniterecharge2020/climber/ClimberCommands.kt]: Usually we like to put sensor statuses in the Subsystem class with something like `val climberAtTop get() = DigitalInput(-1)` and then check that value in the command with `climberAtTop`

**Comment by will-hou** [comment]
We'll also have an encoder on the motor running the climber lift upwards. We should also limit driver control of the arm using that because it's always good to have a backup in addition to the limit switch.

Not sure what the best way to do that would be though. We could measure the encoder ones at the bottom and the top several times and only when run the arm when the `position is < or > +/- a_certain_value`.

**Comment by will-hou** [comment]
This code is certainly....something. But I doubt my brainpower at this time of morning to change it so will leave it **exactly** like how it was tested earlier

**Pull request merged**